plutus-emulator/src/Wallet/Rollup.hs

```diff
@@ -29,7 +29,7 @@ import           Wallet.Rollup.Types      (AnnotatedTx (AnnotatedTx), Beneficial
 
 data TxKey =
     TxKey
-        { _txKeyTxId        :: Digest SHA256
+        { _txKeyTxId        :: BSL.ByteString
```

Comment:
Why not `TxId`?

Reply:
I have no idea. It was already using the underlying type so I thought I'd just leave it since I don't know this code.